backend/substrapp/compute_tasks/outputs.py

```diff
@@ -57,11 +57,13 @@ def _save_performance(self, output: context.OutputResource):
     def _save_models(self, models: list[context.OutputResource]):
         logger.info("saving models")
 
+        # Save models to minio
```

Comment:
It's better to not hardcode the storage name in the comment
```suggestion
        # Save models files
```